src/main/java/frc/robot/subsystems/DrivebaseSubsystem.java

```diff
@@ -8,41 +8,44 @@
 import edu.wpi.first.wpilibj.motorcontrol.PWMVictorSPX;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants;
+import frc.robot.Constants.DriveConstants;
 
 public class DrivebaseSubsystem extends SubsystemBase{
-    private static PWMVictorSPX frontLeft;
-    private static PWMVictorSPX frontRight;
-    private static PWMVictorSPX backLeft;
-    private static PWMVictorSPX backRight;
+    private static CANSparkMax frontLeft;
+    private static CANSparkMax frontRight;
+    private static CANSparkMax backLeft;
+    private static CANSparkMax backRight;
     private static DifferentialDrive m_Drive;
 
     public DrivebaseSubsystem(){
-        frontLeft = new PWMVictorSPX(2);
-        frontRight = new PWMVictorSPX(3);
-        backLeft = new PWMVictorSPX(4);
-        backRight = new PWMVictorSPX(1);
+        frontLeft = new CANSparkMax(Constants.DriveConstants.FRONTLEFT_ID, MotorType.kBrushed);
+        frontRight = new CANSparkMax(Constants.DriveConstants.FRONTRIGHT_ID, MotorType.kBrushed);
+        backLeft = new CANSparkMax(Constants.DriveConstants.BACKLEFT_ID, MotorType.kBrushed);
+        backRight = new CANSparkMax(Constants.DriveConstants.BACKRIGHT_ID, MotorType.kBrushed);
 
-        //restoring factory defaults removes and changes such as inverts and follows so this should alwways be applied before hand
-
-        backLeft.addFollower(frontLeft);
-        backRight.addFollower(frontRight);
+        frontLeft.restoreFactoryDefaults();
+        frontRight.restoreFactoryDefaults();
+        backLeft.restoreFactoryDefaults();
+        backRight.restoreFactoryDefaults();
```

Comment:
When you and Ron were initially implementing the Tank drive on the Flat Stanley, we went over how to build a configureMotors call that included the restoreFactoryDefaults() within its body, and then passed in the Inverted and the Follow states.  That would be a cleaner implementation.